cypress/pages/apply/[resident]/your-situation.ts

```diff
@@ -3,8 +3,8 @@ class ApplyResidentYourSituationPage {
     cy.visit(`/apply/${personId}/your-situation`);
   }
 
-  static getServedInArmedForcesRadioButton() {
-    const testId = 'test-radio-situation-armed-forces.1';
+  static getServedInArmedForcesRadioButton(index: number) {
+    const testId = `test-radio-situation-armed-forces.${index}`;
```

Comment:
Never seen a dot being used in a selector name.
Would guess it's not exactly conventional, but probably not an issue.

Reply:
These controls use index based numbering in the id with `.`, so these test IDs follow the same convention. I agree that it's a bit unusual, but we can tidy those up later.